torch/distributed/tensor/parallel/style.py

```diff
@@ -344,5 +559,33 @@ class ColwiseParallel(ParallelStyle):
     We assume the input to be a replicated :class:`DTensor` and output to be a sharded :class:`torch.Tensor`.
     """
 
-    def __init__(self, _prepare_input=make_input_replicate_1d, _prepare_output=make_sharded_output_tensor) -> None:
-        super().__init__(_prepare_input, _prepare_output)
+    def __init__(
+        self,
+        _prepare_input=None,
+        _prepare_output=None,
+        *,
+        input_layouts=Replicate(),
+        output_layouts=Shard(-1),
+        use_local=True,
```

Comment:
I feel everytime colwise parallel output would be consumed by a rowwise parallel style later (i.e. 3 col -> row, or just col-> row), maybe it's worth making the ColwiseParallel to return DTensor directly? wdyt?

Reply:
Not really, I think for MLP's case we can make the flag to be `False` because for Attn or some cases when DTensor has coverage issue, we always want to lower to tensor ops. If down the road, we are more confident about DTensor CPU overhead and op coverage, we can change this to False as default.